### PR TITLE
Unit & Integration tests

### LINKED ISSUE #103: Unit/Integration tests
Modules:
- [x] Auth
- [x] Depository
- [x] Organization
- [x] Profile
- [x] Project
- [x] User
- [ ] Utils

### PR DESCRIPTION
**Pull request by bmstefanski**
Closes #103

### CODE REVIEW

**Comment by bmstefanski** [comment]
Utils module tests

**Comment by bmstefanski** [comment]
Closes #103

File: reposilite-backend/src/main/java/org/panda_lang/reposilite/profile/ProfileController.java [hunk 1]
```diff
@@ -24,8 +24,8 @@
     @GetMapping("/{project}")
     public ResponseEntity<Map<String, Object>> project(@PathVariable @ApiParam("Account name") String account, @PathVariable @ApiParam("Project name") String project) {
         return ResponseEntity.ok(new LinkedHashMap<String, Object>() {{
```
Comment (dzikoysk) [inline reposilite-backend/src/main/java/org/panda_lang/reposilite/profile/ProfileController.java]: It's generating a new class per each request, performance issue

File: reposilite-backend/src/main/java/org/panda_lang/reposilite/user/UserRegistrationService.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.panda_lang.reposilite.user;
+
+import java.util.Optional;
+
+public interface UserRegistrationService {
+
+    Optional<User> findByUsername(String username);
+
+    User save(User user);
+}
```
Comment (dzikoysk) [inline reposilite-backend/src/main/java/org/panda_lang/reposilite/user/UserRegistrationService.java]: Missing new line after `save` metod

File: reposilite-backend/src/main/java/org/panda_lang/reposilite/utils/validation/FieldMatchValidator.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    public boolean isValid(Object value, ConstraintValidatorContext context) {
+        try {
+            Object firstObj = BeanUtils.getProperty(value, this.firstFieldName);
+            Object secondObj = BeanUtils.getProperty(value, this.secondFieldName);
+
+            return firstObj == null && secondObj == null || firstObj != null && firstObj.equals(secondObj);
+        } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException ex) {
+            ex.printStackTrace();
+        }
+        return true;
```
Comment (dzikoysk) [inline reposilite-backend/src/main/java/org/panda_lang/reposilite/utils/validation/FieldMatchValidator.java]: Missing new line before `return true` statement

File: reposilite-backend/src/test/java/org.panda_lang.reposilite/depository/AbstractDepositoryOwnerBuilderTest.java [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+    }
+
+    private static class TestDepositoryOwner implements DepositoryOwner {
+
+        private String name;
+        private String displayName;
+        private String description;
+        private String email;
+        private String site;
```
Comment (dzikoysk) [inline reposilite-backend/src/test/java/org.panda_lang.reposilite/depository/AbstractDepositoryOwnerBuilderTest.java]: These fields could be final

**Pull request merged**